src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -70,6 +70,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match phone, email and address, but does not match name
```

Comment:
Similar to other comments, the comment describing the test still contains address.